src/components/LocaleSelector/LocaleSelector.jsx

```diff
@@ -1,23 +1,23 @@
 import PropTypes, { func, string } from "prop-types"
-import React from 'react'
-import { Link } from '@material-ui/core';
+import React from "react"
+import { Link } from "@material-ui/core"
 
-const LocaleSelector = ({ locales }) => {
+const LocaleSelector = ({ locales, onSwitch }) => {
   return locales.map(({ icon: Icon, name, url }) => (
-    <Link href={url} key={name} title={name.toUpperCase()}>
+    <Link key={name} title={name.toUpperCase()} onClick={() => onSwitch(name)}>
```

Comment:
It's better to pass handler as ```() => onSwitch(name)``` if you can. In such way link on function doesn't change

Reply:
done